sqlglot/dialects/doris.py

```diff
@@ -684,6 +735,13 @@ def partitionbyrangeproperty_sql(self, expression):
             create_sql = ", ".join(self.sql(e) for e in create_expressions)
             return f"PARTITION BY RANGE ({partition_expressions}) ({create_sql})"
 
+        def partitionedbyproperty_sql(self, expression: exp.PartitionedByProperty) -> str:
+            # Avoid double parens; Doris syntax expects a single set of parens
+            inner = self.sql(expression, "this")
+            if inner.startswith("(") and inner.endswith(")"):
+                inner = inner[1:-1]
+            return f"PARTITION BY ({inner})"
```

Comment:
Why would there be a redundant set of parentheses? What's the AST that results in that SQL?